config_test/evg_settings.yml

```diff
@@ -1,5 +1,5 @@
 database:
-    url: "mongodb://localhost:27017/mci_test"
+    url: "mongodb://localhost:27017"
```

Comment:
Is there a way to avoid making this change? It might only affect me, but this will make tests insert data into the collection I use to run evergreen locally

Reply:
Does the line below not select the db correctly? The reason I have to make this change is to work around https://jira.mongodb.org/browse/GODRIVER-921.